src/layout/Navbar/NavHost.svelte

```diff
@@ -56,53 +61,90 @@
 
 <svelte:window bind:scrollY={y} />
 
-<nav class:scrolled={y > 10}>
-	<a class="menu-btn skiptab-btn" href="#skiptab">Skip navigation</a>
-
-	<button
-		class="menu-btn mobile-only"
-		on:click={() => (menuOpen = !menuOpen)}
-		class:open={menuOpen}
-		aria-label="Menu"
-	>
-		<span class="menu-btn__burger" />
-	</button>
-	<a href="/" id="logo"><img src="/logo.svg" alt="ReVanced Logo" /></a>
-
-	{#key menuOpen}
-		<div
-			class="nav-wrapper"
-			class:desktop-only={!menuOpen}
-			transition:horizontalSlide={{ direction: 'inline', easing: expoOut, duration: 400 }}
+<div id="nav-container">
+	<Query query={pingQuery()} let:data>
+		{#if !data}
+			<span class="banner">
+				<Banner level="caution" permanent>
+					The API is currently unresponsive and some services may not work correctly. Check the <a
+						href="https://status.revanced.app/"
+						target="_blank"
+						rel="noopener noreferrer">status page</a
+					> for updates.
+				</Banner>
+			</span>
+		{/if}
+	</Query>
+
+	<nav class:scrolled={y > 10}>
+		<a class="menu-btn skiptab-btn" href="#skiptab">Skip navigation</a>
+
+		<button
+			class="menu-btn mobile-only"
+			on:click={() => (menuOpen = !menuOpen)}
+			class:open={menuOpen}
+			aria-label="Menu"
 		>
-			<div id="main-navigation">
-				<ul class="nav-buttons">
-					<Navigation href="/" label="Home">Home</Navigation>
-					<Navigation queryKey="manager" href="/download" label="Download">Download</Navigation>
-					<Navigation queryKey="patches" href="/patches" label="Patches">Patches</Navigation>
-					<Navigation queryKey="contributors" href="/contributors" label="Contributors">
-						Contributors
-					</Navigation>
-					<Navigation queryKey={['about', 'team']} href="/donate" label="Donate">Donate</Navigation>
-				</ul>
-			</div>
-			<div id="secondary-navigation">
-				<button on:click={() => (modalOpen = !modalOpen)} aria-label="Settings">
-					<Cog size="20px" color="var(--surface-six)" />
-				</button>
+			<span class="menu-btn__burger" />
+		</button>
+		<a href="/" id="logo"><img src="/logo.svg" alt="ReVanced Logo" /></a>
+
+		{#key menuOpen}
+			<div
+				id="nav-wrapper-container"
+				class:desktop-only={!menuOpen}
+				transition:horizontalSlide={{ direction: 'inline', easing: expoOut, duration: 400 }}
+			>
+				<div id="banner-pad">
+					<Query query={pingQuery()} let:data>
+						<span class="banner">
+							{#if !data}
+								<Banner level="caution" permanent>
+									The API is currently unresponsive and some services may not work correctly. Check
+									the <a
+										href="https://status.revanced.app/"
```

Comment:
Hardcoding the status page may not be the best idea, but there is no other place that links to it. It'll be present in https://api.revanced.app/v4/about under the key "status" in the next API release.

Reply:
If the API is down, how is the website expected to parse that from the API?